conu/backend/docker/container.py

```diff
@@ -179,6 +182,41 @@ def is_port_open(self, port, timeout=2):
             return False
         return check_port(port, host=addresses[0], timeout=timeout)
 
+    def mapped_port_open(self, port, host='0.0.0.0'):
```

Comment:
Could you please elaborate on the use case? I don't understand how does one utilize the function -- I'm pretty sure you have something on your mind -- I just can't imagine how to use it.

Reply:
Also, this can be replaced by check_port method. Added check that mapping exists is achievable by get_port_mappings